docker-compose.yml

```diff
@@ -17,9 +17,11 @@ services:
     env_file:
       - ./sites/public/.env
     environment:
-      # TODO(CityOfDetroit/affordable-housing-app#84): login currently doesn't work, likely
-      # needs a separate env variable to point client requests to localhost:3100 instead.
+      # The URLs are different here because requests made using BACKEND_API_BASE are done from
+      # the NextJS server, which must address other containers by container name. Requests made
+      # using INCOMING_HOOK_BODY are done from the client side browser and must use localhost.
       BACKEND_API_BASE: "http://backend-core:3100"
+      INCOMING_HOOK_BODY: "http://localhost:3100"
```

Comment:
I'm skimming https://docs.netlify.com/configure-builds/environment-variables/#build-hook-metadata-and-payload and https://docs.netlify.com/configure-builds/build-hooks/ to try to understand what this is doing: there's a POST request happening somewhere, and the body of that POST is now "http://localhost:3100". Is there anything on the other end, receiving the POST request, that knows what to do with that message body? (i.e. somewhere where we say "whatever was in the body of the request, swap that in for BACKEND_API_BASE"?)

Reply:
This change isn't really related to the Netlify variable itself or what that does when deploying it on Netlify; just the fact that it's being used here https://github.com/CityOfDetroit/bloom/blob/main/sites/public/next.config.js to override the BACKEND_API_BASE only for ui-components. Which, I think is needed when deploying this for the exact reason that we need it here (client side logic needing a different API address).

Reply:
Grrr, Github search failed me: https://screenshot.googleplex.com/4HgfQyQVcy8aLpG

If I'd've seen that we were consuming INCOMING_HOOK_BODY somewhere, I wouldn't have gone off on my wild goose chase to figure out what that variable was. Oh well, thanks for pointing to `next.config.js`!

(and yes, this is enough evidence that I should just use VSCode to search 🙃 )